jsIndex.js

```diff
@@ -0,0 +1,59 @@
+
+var playerScore = 0;
+var compScore = 0;
+function makeid() {
+  var text = "";
+  var possible = "rps";
+    text += possible.charAt(Math.floor(Math.random() * possible.length));
+  return text;
+}
+//The computer randomly gets a character r, p or s relating to rock paper or
+//scissors to compare against the players choice.
+function displayScore(){
+  alert("Player Score - " + playerScore + " -- " + compScore + " - Computer Score" );
```

Comment:
I would include here what happened e.g. "You picked scissors, the computer picked rock. Rock crushes scissors!" Or something like that. It's a bit abrupt, otherwise.